gradlew.bat

```diff
@@ -64,28 +64,14 @@ echo location of your Java installation.
 
 goto fail
 
-:init
-@rem Get command-line arguments, handling Windows variants

```

Comment:
Does this make it so we can't build on Windows? Since this is OSS I'd like to keep it easy to develop against every platform.

Reply:
We can still build on Windows because we still have a `gradlew.bat` file. This was just part of the update to the new gradle version: new gradle versions often include an updated gradle wrapper, and the gradle wrapper is something we version control with the rest of our code.